string map_name
string type     # 1:3d 2:2d
uint32 seq

string action   # start finish cancel stitch

string map1_name
string map2_name
---

string result
